shared/gh/js/views/gh.visibility.js

```diff
@@ -61,6 +157,8 @@ define(['gh.core'], function(gh) {
      * @private
      */
     var addBinding = function() {
+        // Initialise the visibility button
+        $(document).on('gh.part.changed', initialiseVisibilityButton);
```

Comment:
Can you rename it to `setUpVisibilityButton` to be consistent with our other setup functions?